docs/arch/adr-001-send-unpublish-emails-for-single-pages.md

```diff
@@ -0,0 +1,61 @@
+# Decision record: reinstate unpublish emails for single page subscriptions
```

Comment:
I'd suggest you put this in Email Alert API where there are various other Email Alert related ADRs: https://github.com/alphagov/email-alert-api/tree/main/docs/adr

I imagine this could fit in either one of these apps (I myself don't really think they should be separate repos) but it's easier to think of this as a sidecar of Email Alert API rather than something of equal importance.

Reply:
Cool, I'll do that today